recipes/mariadb/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "mariadb" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mariadb-{{ version }}.zip
+  sha256: 73a0acbd2ace381ec1bcfc61ced7a79a519e66c02c24e12ae6d27baa03717876
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
```

With this not being noarch, the Python dependency should be unversioned:
```suggestion
    - python
    - pip
  run:
    - python
```